pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/wheel/WheelCacheTest.groovy

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.wheel
+
+import com.linkedin.gradle.python.extension.PlatformTag
+import com.linkedin.gradle.python.extension.PythonVersion
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import static com.linkedin.gradle.python.extension.PythonTag.C_PYTHON
+
+class WheelCacheTest extends Specification {
+
+    static final PythonVersion PYTHON_33 = version('3.3')
+    static final PythonVersion PYTHON_35 = version('3.5')
+
+    @Rule
+    TemporaryFolder temporaryFolder
+
+    static final PlatformTag MAC_TAG = new PlatformTag('macosx_10_10_x86_64')
+    static final PlatformTag LINUX_TAG = new PlatformTag('linux_x86_64')
+
+    @Unroll
+    def 'will find wheels #wheelName == #pythonVersion'() {
+        given:
+        temporaryFolder.newFile('lib-1.2-' + wheelName + ".whl")
+
+        def cache = new WheelCache(temporaryFolder.root, pythonTag, platformTag)
+
+        expect:
+        cache.findWheel('lib', '1.2', pythonVersion).isPresent() == matches
+
+        where:
+        wheelName                        | pythonVersion | pythonTag | platformTag | matches
+        'cp33-cp33m-linux_x86_64'        | PYTHON_33     | C_PYTHON  | LINUX_TAG   | true
```

Comment:
We don't use Python 3.3 and it's not supported any more.
3.4 is probably have the same destiny too.
We use 3.5 and 3.6 internally.
Could we make test examples with Python 3.5 and 3.6 only?

Reply:
Those values came from https://www.python.org/dev/peps/pep-0425/#id1. I would prefer to use the values from the PEP. We also need to make sure we have different major versions.

Reply:
OK. Can we add some Python 2.7 tests here then. Notice that the format will be `macosx-10.12-intel` instead of `macosx-10.10-x86_64` for the command used to produce that string. We need to check if there's the similar difference on Linux.